src/demos/MessageReply/MessageReply.md

```diff
@@ -0,0 +1,49 @@
+```js
+const { AvatarSize } = require('../../components/Avatar');
+const { FixedGridRow, FixedGridColumn } = require('../../components/FixedGrid');
+const { GutterSize } = require('../../components/Block');
+const { IconSize } = require('../../components/Icon');
+const { TextColor, TextSize } = require('../../components/Text');
+
+
+const user1 = {
+  image: 'user.png',
+  name: 'Sally Super-super-super-long-middle-name Smilefields',
+};
+const user2 = {
+  image: 'user.png',
+  name: 'Tiny Tim',
+};
+
+<FixedGridRow gutterSize={GutterSize.LARGE}>
+  <FixedGridColumn fixed={true} width={32}>
+    <Avatar imageUrl={user1.image} name={user1.name} size={AvatarSize.SMALL} />
+  </FixedGridColumn>
+  <FixedGridColumn>
+    <Block bottomSpacing={GutterSize.XSMALL} className="byline">
+      <Text maxWidth="100%">
+        <Text bold={true}>{user1.name}</Text> <Text color={TextColor.METADATA} size={TextSize.MEDIUM_SUB}>in reply to</Text> <Text bold={true}>{user2.name}</Text>
+      </Text> <Text color={TextColor.METADATA} size={TextSize.MEDIUM_SUB}> - 5 hours ago from Desktop</Text>
+    </Block>
+    <Block bottomSpacing={GutterSize.MEDIUM} className="message">
+      hey that's a great idea! let's discuss this further at the monthly offsite next week
+    </Block>
+    <Block textSize={TextSize.SMALL} className="actions">
+      <HorizontalList>
+        <HorizontalListItem>
+          <Clickable><Icon icon="like" size={IconSize.XSMALL} /> Like</Clickable>
+        </HorizontalListItem>
+        <HorizontalListItem>
+          <Clickable><Icon icon="edit" size={IconSize.XSMALL} /> Reply</Clickable>
+        </HorizontalListItem>
+        <HorizontalListItem>
+          <Clickable><Icon icon="share" size={IconSize.XSMALL} /> Share</Clickable>
+        </HorizontalListItem>
+        <HorizontalListItem>
+          <Clickable>...</Clickable>
+        </HorizontalListItem>
+      </HorizontalList>
+    </Block>
+  </FixedGridColumn>
+</FixedGridRow>
```

Comment:
Why not move all this into the `.tsx` file?

Reply:
good question. this way the full example is available for people to play with in the code panel.  would be nice if we didn't have to include a dummy tsx file

Reply:
I understand. The code for `Hovercard` demos currently lives in the `.tsx` file though...

Reply:
agree would be good to be consistent. let's think this through when we improve the examples/sections format